sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -101,7 +101,7 @@ var client = new FormRecognizerClient(new Uri(endpoint), new DefaultAzureCredent
 `FormRecognizerClient` provides operations for:
 
  - Recognizing form fields and content, using custom models trained to recognize your custom forms.  These values are returned in a collection of `RecognizedForm` objects. See example [Recognize Custom Forms](#recognize-custom-forms).
- - Recognizing form content, including tables, lines and words, without the need to train a model.  Form content is returned in a collection of `FormPage` objects. See example [Recognize Content](#recognize-content).
+ - Recognizing form content, including tables, lines, words, and selection marks like radio buttons and check boxes without the need to train a model.  Form content is returned in a collection of `FormPage` objects. See example [Recognize Content](#recognize-content).
```

Comment:
I wonder if we should say something about training custom models with labels to recognize selection marks as well

Reply:
I thought about it but looking at `Recognizing form fields and content, using custom models trained to recognize your custom forms.` makes me think that it already includes it. Although `selectionmarks` is a new concept, the idea of recognizing fields and content still applies.